.changelog

```diff
@@ -4,6 +4,9 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Remove package sync paragraph in package-management doc since
+  it's not available for salt clients and traditional is no
+  longer supported (bsc#1221279)
```

Comment:
```suggestion
- Remove package sync paragraph in package-management doc since
  it is not available for Salt clients and traditional clients are no
  longer supported (bsc#1221279)
```